druid/src/window.rs

```diff
@@ -402,14 +402,14 @@ impl<T: Data> Window<T> {
         }
         self.root
             .set_origin(&mut layout_ctx, data, env, Point::ORIGIN);
+        // This lifecycle call will call post_event_processing for us.
         self.lifecycle(
             queue,
             &LifeCycle::Internal(InternalLifeCycle::ParentWindowOrigin),
             data,
             env,
-            false,
+            true,
```

Comment:
is this related to the fix, or is this just you doing some cleanup while you're in here?

Reply:
It was cleanup because it was calling post_event_processing twice (once inside the lifecycle, once outside). 
I don't think it is necessary for the fix, I just noticed it.

Reply:
@rjwittams can we revert this, and have this patch only include the fix? I'd like to have as few moving parts as possible, so we can be confident about the root cause.

I will open an issue about improving post-event stuff.

Reply:
Done